### PR TITLE
Add extension for date formats

### PR DESCRIPTION
**Pull request by jvc9109**
add extension for properly format dob and simple dates in php-sdk

### CODE REVIEW

**Review by justnero — APPROVED** [review]

File: openapi/components/schemas/AML.yaml [hunk 1]
```diff
@@ -119,6 +119,7 @@ properties:
       description: Date of birth.
       format: date
       example: 1706-01-17
+      x-simple-date-format: true
```
Comment (ashkarpetin) [inline openapi/components/schemas/AML.yaml:122]: What are the possible formats for `format: date`? I don't clearly understand the purpose of that extension.

Comment (jvc9109) [reply openapi/components/schemas/AML.yaml:122]: There are two in the composition of the Models. A full datetime value like `2019-10-12T07:20:50.52Z` and a simple format like the Date of birth. This two formats can conflict in the json serialization of the php-sdk autogenerated.

Comment (ashkarpetin) [reply openapi/components/schemas/AML.yaml:122]: Are not able to distinguish `format:date` and `format:date-time` during autogeneration?

Comment (justnero) [reply openapi/components/schemas/AML.yaml:122]: Good point. @jvc9109 do we have a `date-time` set consistently in other places?

Comment (jvc9109) [reply openapi/components/schemas/AML.yaml:122]: They both are mapped to `DatetimeImmutable` before template rendering or postprocessing. (Those steps are the ones we can have more control and customization)

**Comment by jvc9109** [comment]
Closing PR to implement an alternative. Thanks @ashkarpetin  for the suggestions!

**Pull request closed without merging**